src/router/Router.js

```diff
@@ -164,6 +167,18 @@ export default class Router extends DisposableBase {
             return this._observeEventsUsingConventions(modelId, object, methodPrefix);
         }
     }
+    addOnErrorHandler(handler) {
+        this._onErrorHandlers.push(handler);
```

Comment:
it's not the end of the world. The call to this handler is in a try catch so we should be ok